### PR TITLE
pkgdown skeleton and gh action

### PR DESCRIPTION
**Pull request by edavidaja**
I'll do vignettes in a separate branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ rscpages: Curate your content on RStudio Connect
 
 <!-- badges: start -->
 
+[![CRAN status](https://www.r-pkg.org/badges/version/rscpages)](https://CRAN.R-project.org/package=rscpages)
```
Comment (Lytol) [inline README.md:6]: You want a CRAN badge before we submit?

**Review by Lytol — APPROVED** [review]
:+1:

File: README.md (hunk 1 above)
Comment (edavidaja) [reply README.md:6]: Yep, at least one person during testing missed that it wasn't on cran, so the badge will show "not published" until it lands, at which point it should update.

**Comment by Lytol** [comment]
Gooooooood call!

**Pull request merged**